### ISSUE TITLE
FIshtown.M2 KStream/KTable and 0.11*+ Kafka Brokers

### ISSUE THREAD

**Issue by estelle-peterson** [spring-cloud/spring-cloud-stream-starters#51]
Hey guys,

I'd just like to confirm I'm reading this right in that Fishtown.M2 KStream/KTable APIs work with 0.11* or later versions of kafka brokers?

https://github.com/spring-cloud/spring-cloud-stream-starters/releases/tag/vFishtown.M2

**Comment by sobychacko** [comment]
@estelle-peterson That is right. (Please use GH only for new feature requests or bug reports - questions like these can be asked and answered on gitter - https://gitter.im/spring-cloud/spring-cloud-stream)
